openapi/components/schemas/Applications/Application.yaml

```
@@ -74,7 +74,17 @@ properties:
       - GetInvoiceCollection
       - GetCustomer
       - GetCustomerCollection
-
+  configurationUrl:
+    description: External URL to visit in order to configure the application during installation.
+    type: string
+    format: url
+    nullable: true
+  configuredBy:
+    type: string
```

issue: we should describe all fields (I'm not sure if my suggestions are correct here)
```suggestion
    type: string
    description: Name of the user who configured the application.
```
or
```suggestion
    type: string
    description: Name of the user who installed the application.
```

---

This field determines who must configure the application - the application `user` or the application owner (`application` itself) and not the name of a particular user

---

Updated based on your input. We probably don't need to state `user` or `application` in the description, because they are listed as enums